QueueManagerTest/SearchManagerTests.cs

```diff
@@ -80,5 +80,10 @@ public void TestSearchMultipleItemsWhenASpaceIsPresentWihTwoWords()
             List<string> listToSearch = new List<string>() { "The word", "alpha add", "beta sdf" };
             Assert.AreEqual(new List<string>() { "The word" }, _searchManager.SearchFor(listToSearch, "the wo"));
         }
+        [TearDown]
```

Comment:
It's possible that this will never need a tear down. If someone decides they need a teardown method they can implement it. I don't think anyone would forget to add a teardown if they specifically needed to do something to clean up the test. I say dump it. TODO also doesn't fit since there isn't any task here.